pkg/reconciler/eventlistener/resources/container.go

```diff
@@ -49,21 +49,19 @@ func MakeContainer(el *v1beta1.EventListener, configAcc reconcilersource.ConfigA
 
 	ev := configAcc.ToEnvVars()
 
-	containerSecurityContext := corev1.SecurityContext{}
+	containerSecurityContext := corev1.SecurityContext{
+		AllowPrivilegeEscalation: ptr.Bool(false),
+		Capabilities: &corev1.Capabilities{
+			Drop: []corev1.Capability{"ALL"},
+		},
+		RunAsNonRoot: ptr.Bool(true),
+		SeccompProfile: &corev1.SeccompProfile{
+			Type: corev1.SeccompProfileTypeRuntimeDefault,
+		},
+	}
 	if *c.SetSecurityContext {
```

Comment:
We are already setting context now irrespective of flag. I believe we need another flag to enable/disable user in security context.

Reply:
Ah i see what you mean 

then we may need to remove the existing flag `el-security-context`  and add new one `el-set-user-security-context` ??

Reply:
No need to remove the older one. Just add a newer one for userid.

Reply:
then what is the use of `el-security-context` ?

Reply:
After thinking about upgrade and allel-security-context